cosmos/operators/local.py

```diff
@@ -816,3 +816,12 @@ def __init__(self, **kwargs: str) -> None:
         raise DeprecationWarning(
             "The DbtDepsOperator has been deprecated. " "Please use the `install_deps` flag in dbt_args instead."
         )
+
+
+class DbtSourceLocalOperator(DbtLocalBaseOperator):
+    """
+    Executes a dbt source freshness command.
+    """
+
+    ui_color = "#34CCEB"
+    base_cmd = ["source", "freshness"]
```

Comment:
What do you think if we added freshness to `template_fields`?

This way, users would be able to visualise the freshness values in the UI, similar to other ones:

![cosmos_rendered_templates](https://github.com/user-attachments/assets/b13efdb1-ae15-4319-997f-bf09968929b2)

Example of how to do:
https://github.com/astronomer/astronomer-cosmos/blob/e61f3a365f6c3a0b550cda298516f64ad1321189/cosmos/operators/local.py#L113

Reply:
![image](https://github.com/user-attachments/assets/c11c7807-295d-4067-aef6-81b4b9a408c0)
The latest PR added freshness like the picture.
It is adding the entire [sources.json file](https://docs.getdbt.com/reference/artifacts/sources-json#:~:text=This%20file%20contains%20information%20about,power%20its%20Source%20Freshness%20visualization.)

I couldn't find where dbt was compiling the freshness sql queries, it probably doesn't store it locally